client/src/containers/FormAndEventsWrapper.js

```diff
@@ -27,16 +28,28 @@ class FormAndEventsWrapper extends React.Component {
     event.preventDefault();
   }
 
+  handleAltSubmit(event) {
```

Comment:
See if you can have a go at sharing the existing `handleSubmit()` function instead of creating this new one :-)

Reply:
Did you ever look at this?